Misc/NEWS.d/next/Library/2024-05-26-22-22-51.gh-issue-119594.fnQNM8.rst

```diff
@@ -0,0 +1 @@
+Fraction.__pow__ accepts (but does not implement) optional modulo argument.
```

Comment:
If one calls pow(fractions.Fraction, x, module) with modulo not None, the error message now says that the types are incompatible rather than saying pow only takes 2 arguments.  Patch by Wim Jeantine-Glenn and Mark Dickinson.